src/inference/dev_api/openvino/runtime/threading/cpu_map_scheduling.hpp

```diff
@@ -35,4 +35,16 @@ std::vector<std::vector<int>> apply_hyper_threading(bool input_type,
                                                     const bool input_changed,
                                                     const std::vector<std::vector<int>>& proc_type_table);
 
+/**
+ * @brief      Update CPU pinning property according to platform and user setting
+ * @param[in]  input_pinning indicates value of property enable_cpu_pinning.
+ * @param[in]  input_changed indicates if value is set by user.
+ * @param[in]  stream_info_table indicates the stream details that will be created..
+ * @param[in]  original_proc_type_table indicates CPU details on this platform
+ * @return     updated proc_type_table which removed unmatched processors
+ */
+bool update_cpu_pinning(const bool input_pinning,
```

Comment:
as I see, this function is not used currently. Why do we need it then?
Secondly, it's not exported. So, it cannot be used outside of OpenVINO library. For example CPU plugin cannot use it, so does it really need to be in API folder?

Reply:
Move to CPU plugin folder as internal API.

Reply:
but can it be useful for GPU as well?
is it used in implementation of `enable_thread_pinning` ?

Reply:
This is the last step of new flow. GPU can use it as well. Move to `threading` folder.